oshc/main/static/main/css/app.css

```diff
@@ -43,10 +43,11 @@ a {
 
 .navbar-inverse {
   background-color: #24292e;
+  border-color: #24292e;
 }
 
-.navbar-inverse {
-  border-color: #24292e;
+.navbar-inverse .navbar-nav>li>a {
```

Comment:
change `.navbar-inverse .navbar-nav>li>a {` to `.navbar-inverse .navbar-nav > li > a {`